examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    today=pk.epoch(8335.5, "mjd2000")
</pre><p>maybe nice for non astro people to have a link to explain what this is</p><p> </p><p>maybe https://esa.github.io/pykep/documentation/core.html#pykep.epoch ?</p><p>Ah btw, you can also just use <code>pykep.epoch_from_string('2002-01-20 23:59:54.003')</code> that may be easier to understand? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>